cmd/integration_test/build/buildspecs/quick-test-eks-a-cli.yml

```diff
@@ -130,7 +164,7 @@ phases:
           BUNDLES_OVERRIDE=true
         fi
       - SKIPPED_TESTS=$(yq e ".skipped_tests | @csv" ${CODEBUILD_SRC_DIR}/test/e2e/SKIPPED_TESTS.yaml)
-      - QUICK_TESTS="($(yq e '.quick_tests | join("|")' ./test/e2e/QUICK_TESTS.yaml))"
+      - QUICK_TESTS="($(yq e '.quick_tests | map("^" + . + "$") | join("|")' ./test/e2e/QUICK_TESTS.yaml))"
```

Comment:
It's not convenient that a test case regular expression spread across quick-test-eks-a-cli.yml and quick_tests.yaml. Can we put the "^" "$" symbols in the QUICK_TESTS.yaml?

Reply:
I understand your concern about the test case regular expression being spread across different files.
 
 I can make the change to include it in the `QUICK_TEST.yaml` itself